### ISSUE TITLE
Gerald's air assist has come apart

### ISSUE THREAD

**Issue by JackiePease** [DoESLiverpool/somebody-should#1476]
Gerald's air assist isn't attached. There doesn't seem to be any obvious way to join the 2 parts together - maybe someone has a photo?

I was still able to use the laser cutter but it's important to keep a close eye on it as there's a higher risk of fire.

**Comment by JackiePease** [comment]
![IMG_20200812_193104](https://user-images.githubusercontent.com/12802256/90054498-dc45e600-dcd3-11ea-81d8-a96afc99ee60.jpg)

**Comment by JackiePease** [comment]
I spoke to Jade this morning, she's just been reconnecting the 2 ends, and says it's happened several times. Is it supposed to have a cable tie or similar?

**Comment by JackiePease** [comment]
Spoke to @amcewen - this is a useful weak spot as, if it fails here, it stops the tube from dragging across the bed and moving the work. Apparently there are plans to change the whole top part.  
Should we have a sign to ask people to check the air assist?

**Comment by magman2112** [comment]
It appears that the silicone tubing that was on the air assist last time I worked on Gerald has gone missing, possibly damaged at some time. The PVC tubing that had replaced it is a little larger than the nipple on the laser head as it was a little too large diameter, so would come loose. We don’t appear to have any spare silicone tubing left.

I found some thinner PVC tubing in the laser spares cupboard, that I was able to fit to the laser head with a little heat, which is a good tight fit. This should do for now to fix the situation. I will though find and buy some replacement silicone tubing, as the whole air assist tubing run is due for replacement. 

I believe we need a 5mm internal diameter for this tubing, with a wall size that is not too thick to fit through the drag chains.

I will close this issue for now though, as the replacement tube is already highlighted in issue #1443.
